File: openshift-examples/X-Polyglot-Examples/src/main/js/MyHttpServer.js
vertx.createHttpServer()
    .requestHandler(function (req) {
        req.response().putHeader("content-type", "text/html").end("<h1>Hello from JavaScript</h1>")
    })
    .listen(8080, function(res, err) {
        if (err) {
            console.log("Unable to start the HTTP server: " + err.getMessage());
        } else {
            console.log("Server started on port " + res.actualPort());
        }
    });

File: openshift-examples/X-Polyglot-Examples/src/main/ruby/MyHttpServer.rb
$vertx.create_http_server.request_handler { |req|
  req.response.put_header("content-type", "text/html").end("<html><body><h1>Hello from vert.x!</h1></body></html>")
}.listen(8081) { |err, svc|
  if err == nil
    puts "HTTP server started on port #{svc.actual_port}"
  else
    puts "Unable to start HTTP server: #{err.get_message}"
  end
}
